api_app/analyzers_manager/migrations/0058_analyzer_config_ip2location.py

```diff
@@ -0,0 +1,166 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "name": "ip2location",
+    "python_module": {
+        "module": "ip2location.Ip2location",
+        "base_path": "api_app.analyzers_manager.observable_analyzers",
+    },
+    "description": "IP2Location.io allows users to check IP address location in real time.",
```

Comment:
can you please add a link to the service main page?

Reply:
Added